cypress/integration/awesome_bar.js

```diff
@@ -10,34 +10,34 @@ context('Awesome Bar', () => {
 	});
 
 	it('navigates to doctype list', () => {
-		cy.get('#navbar-search').type('todo', { delay: 200 });
-		cy.get('#navbar-search + ul').should('be.visible');
-		cy.get('#navbar-search').type('{downarrow}{enter}', { delay: 100 });
+		cy.findByPlaceholderText('Search or type a command (Ctrl + G)').type('todo', { delay: 200 });
+		cy.get('.awesomplete').findByRole('listbox').should('be.visible');
+		cy.findByPlaceholderText('Search or type a command (Ctrl + G)').type('{downarrow}{enter}', { delay: 100 });
```

Comment:
```suggestion
		cy.findByPlaceholderText(/Search or type a command/).type('{downarrow}{enter}', { delay: 100 });
```

Most text searches support regex -> easy partial text matching.

Reply:
Can be used in the case of variable text in a string.